XamarinCommunityToolkit/Core/StreamMediaSource.shared.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Core
+{
+	public class StreamMediaSource : MediaSource, IStreamImageSource
+	{
+		readonly object synchandle = new object();
+		CancellationTokenSource cancellationTokenSource;
+
+		TaskCompletionSource<bool> completionSource;
+
+		public static readonly BindableProperty StreamProperty = BindableProperty.Create(nameof(Stream), typeof(Func<CancellationToken, Task<Stream>>), typeof(StreamMediaSource),
+			default(Func<CancellationToken, Task<Stream>>));
```

Comment:
```suggestion
		public static readonly BindableProperty StreamProperty 
			= BindableProperty.Create(nameof(Stream), typeof(Func<CancellationToken, Task<Stream>>), typeof(StreamMediaSource));
```

Reply:
We can skip passing default(Func<CancellationToken, Task<Stream>>) cuz it's null. Null is a default value for "defaultValue" parameter.